kuma/users/jinja2/users/includes/user_revision_history_form.html

```diff
@@ -42,25 +42,36 @@
                 <tr>
                     <td></td>
                     <td colspan=4>
+                        {# If this is a revision, compare with most recent revision #}
                         {% if previous_id %}
                             {% with revision_from=revision.previous, revision_to=revision %}
                               <div class="dashboard-compare">
                                 {% include "wiki/includes/revision_diff_table.html" %}
                               </div>
                             {% endwith %}
-                        {% endif %}
+
+                        {# If this is a new translation, compare it with English translation #}
+                        {% elif revision.document.locale != 'en-US' %}
+                            {% with revision_from=revision.document.parent.current_revision, revision_to=revision %}
```

Comment:
Instead of ``revision_from=revision.document.parent.current_revision``, use ``revision_fromrevision.based_on``, which is the English's documents revision at the time of translation.